kili/mutations/asset/helpers.py

```diff
@@ -77,16 +79,20 @@ def process_content(input_type: str,
     Process the array of contents
     """
     if input_type in ['IMAGE', 'PDF']:
-        return [content if is_url(content) else (content
-            if (json_content_array is not None and json_content_array[i] is not None)
-            else (encode_base64(content) if check_file_mime_type(content, input_type) else None))
-            for i, content in enumerate(content_array)]
+        return [content if is_url(content)
+                else (content
+                      if (json_content_array is not None and json_content_array[i] is not None)
+                      else (encode_base64(content) if check_file_mime_type(content, input_type)
+                            else None))
+                for i, content in enumerate(content_array)]
```

Comment:
In order to avoid those types of formatting modifications which do not change anything, I would recommend black formatter

Reply:
Yes indeed, he have a ticket for that that we haven't taken yet: https://linear.app/kili-technology/issue/ML-33/aakd-i-want-the-python-sdk-code-formatted-with-black